install.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
-
 cli_url=https://github.com/ingrammicro/concerto/raw/master/binaries/concerto.x64
```

Comment:
From my understanding, this should be changed, because of binaries should not be available anymore.

Something like this should replace it : https://github.com/ingrammicro/concerto/releases/download/{LATEST_RELEASE}/concerto.amd64.linux

I'm right?

Reply:
Yes, but I'm not sure if it will be easy to get the latest release in the script... maybe we should remove the install.sh...